pkg/controller/mongodb/mongodb_controller.go

```diff
@@ -132,17 +132,46 @@ func (r ReplicaSetReconciler) ensureAutomationConfig(mdb mdbv1.MongoDB) error {
 }
 
 func buildAutomationConfig(mdb mdbv1.MongoDB) automationconfig.AutomationConfig {
-	domain := getDomain(mdb.ServiceName(), mdb.Namespace, mdb.Name)
+	domain := getDomain(mdb.ServiceName(), mdb.Namespace, "")
```

Comment:
maybe good for now, but we many need a TODO here

![](https://media.giphy.com/media/NEJCPYzr6xmAU/giphy.gif)

Reply:
Actually not, because that parameter is the "cluster domain"; when passed the empty string, then the default one will be used ("cluster.local")